Sources/TuistKit/Services/TestService.swift

```diff
@@ -170,20 +170,20 @@ final class TestService {
             ) {
                 try FileHandler.shared.createFolder(cacheDirectoriesProvider.cacheDirectory(for: .tests))
```

Comment:
This looks like it belongs outside the conditional as well. Since the directory is still being created without it being executed, it might be redundant code.

Reply:
That is correct, there might be instances where `tuist test MyScheme` would fail because of this.

Reply:
Good catch, I'll move that 👍

Reply:
Should be good to go now 👍